### PR TITLE
fixed cli help commands, minor jsonrpc api refactor

### PR DESCRIPTION
**Pull request by lyoshenka**

### CODE REVIEW

**Comment by kauffj** [comment]
:clap: :clap: :clap:

**Comment by kauffj** [comment]
wait, this may not be what I thought it was... maybe only one clap was appropriate

**Comment by lyoshenka** [comment]
yea that's far too many claps for what i did

File: lbrynet/lbrynet_daemon/auth/client.py [hunk 1]
```diff
@@ -35,26 +34,21 @@ def __init__(self, key, timeout, connection, count, service, cookies, auth, url,
...
-                             url=self.__url,
-                             login_url=self.__service_url)
-
-    def __call__(self, *args):
+            raise AttributeError  # Python internal stuff
+
+        def f(*args):
+            return self.call(name, args)
+
+        return f
```
Comment (jobevers) [inline lbrynet/lbrynet_daemon/auth/client.py]: I think you can do `return functools.partial(self.call, name)`

**Comment by jobevers** [comment]
Is this still in something being worked on? The last commit message "before getting on plane" makes it sound like there is perhaps more to come.

**Comment by lyoshenka** [comment]
comment from jack:

have you tried lbrynet-cli with and without auth, with functions with and without params?

and did the change to `call` from `__call__` introduce any bugs?

**Comment by lyoshenka** [comment]
@kauffj want to finish this?

**Comment by lyoshenka** [comment]
i'm not gonna get to this for at least a week, maybe more. this is not that important atm

**Comment by lyoshenka** [comment]
totally gonna finish this next week

**Pull request closed without merging**